firmware/quadruna/VC/src/app/app_globals.h

```diff
@@ -1,8 +1,29 @@
 #pragma once
 
+#include <stdbool.h>
+#include "app_heartbeatMonitor.h"
+#include "io_led.h"
+#include "io_buzzer.h"
+#include "app_timer.h"
+
+#define BUZZER_ON_DURATION_MS 2000
+
+typedef struct
+{
+    const Buzzer *const buzzer;
```

Comment:
create a function in io_buzzer called io_buzzer_init and pass this in, and store it as a static variable.